tests/sample.csv

```diff
@@ -1,3 +1,3 @@
-﻿Peptide Sequence,Modification Type,Modification Position,MHC Name
+Peptide Sequence,Modification Type,Modification Position,MHC Molecule
```

Comment:
I think the MHC Molecule column should go first, as it does in the HTML form.

Reply:
Commit 82dd9e5